playbook/app/pb_kits/playbook/pb_advanced_table/Components/SortIconButton.tsx

```diff
@@ -4,28 +4,47 @@ import { Header } from "@tanstack/react-table"
 import { GenericObject } from "../../types"
 
 import Icon from "../../pb_icon/_icon"
+import { getAllIcons } from "../../utilities/icons/allicons"
 
 import { displayIcon } from "../Utilities/IconHelpers"
 
 type SortIconButtonProps = {
   header: Header<GenericObject, unknown>
   sortIcon?: string | string[]
 }
+
 export const SortIconButton = ({ header, sortIcon }: SortIconButtonProps) => {
 
+const firstIcon = displayIcon(sortIcon)[0] 
+const secondIcon = displayIcon(sortIcon)[1] 
+const upIcon = getAllIcons()["arrowUpShortWide"].icon as unknown as { [key: string]: SVGElement }
+const downIcon = getAllIcons()["arrowDownShortWide"].icon as unknown as { [key: string]: SVGElement }
+
   return (
     <>
       {header.column.getIsSorted() === "desc" ? (
         <div className="sort-button-icon" 
-            key={displayIcon(sortIcon)[0]}
+            key={firstIcon}
         >
-          <Icon icon={displayIcon(sortIcon)[0]} />
+        { firstIcon === "arrow-up-short-wide" ? (
+          <Icon 
+              className="svg-inline--fa"
```

Comment:
svg-inline--fa is a classname that has styles we need should probably rename this is yucky